packages/ui/src/components/ProjectCard/ProjectCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Box, Flex, forwardRef, Grid } from '@chakra-ui/react';
+
+import { statusValueCheck } from '../../helpers/statusValueCheck';
+import { Button } from '../Button/Button';
+import { Typography } from '../Typography/Typography';
+
+type DoneProps = {
+  status: 'done';
+  points: number;
+  pointsMax: number;
+  date?: never;
+};
+
+type NotDoneProps = {
+  status: 'idle' | 'doing' | 'review';
+  date: Date;
+  points?: never;
+  pointsMax?: never;
+};
+
+type CommonProps = {
+  image: string;
+  title: string;
+  url: string;
+};
+
+export type ProjectCardProps = CommonProps & (DoneProps | NotDoneProps);
+
+export const ProjectCard = forwardRef<ProjectCardProps, 'div'>(
```

Comment:
🤔 I looked on the card in the Storybook and it doesn't seem right. The content of the card should fill the entire space instead of being centered.
<img width="427" alt="CleanShot 2021-07-25 at 10 36 55@2x" src="https://user-images.githubusercontent.com/22938071/126893041-3d854adb-94b1-4173-8d42-5c64f76d5024.png">